File: dbpro/src/main/java/kr/ac/sejong/feat/actor/ActorDTO.java
package kr.ac.sejong.feat.actor;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import kr.ac.sejong.config.ColumnName;

import java.sql.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ActorDTO {
	@ColumnName("ACTOR_CODE")
	private String actorCode;

	@ColumnName("ACTOR_NAME")
	private String actorName;

	@ColumnName("ACTOR_GENDER")
	private String actorGender;

	@ColumnName("ACTOR_BIRTH")
	private Date actorBirth;
}

File: dbpro/src/main/java/kr/ac/sejong/feat/actor/ActorDTO.kt
package kr.ac.sejong.feat.actor

import kr.ac.sejong.config.ColumnName
import java.sql.Date

data class ActorDTO(
    @ColumnName("ACTOR_CODE")
    private val actorCode: String? = null,

    @ColumnName("ACTOR_NAME")
    private val actorName: String? = null,

    @ColumnName("ACTOR_GENDER")
    private val actorGender: String? = null,
    
    @ColumnName("ACTOR_BIRTH")
    private val actorBirth: Date? = null
)